client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcConnection.java

```diff
@@ -500,7 +500,8 @@ public void testConcurrentCancellationOnStatementClose()
         }
     }
 
-    @Test(timeOut = 60_000, dataProviderClass = DataProviders.class, dataProvider = "trueFalse")
+    // TODO https://github.com/trinodb/trino/issues/10096 - enable test once concurrent jdbc statements are supported
+    @Test(timeOut = 60_000, dataProviderClass = DataProviders.class, dataProvider = "trueFalse", enabled = false)
```

Comment:
or keep enabled, but `@Flaky`?

Reply:
Based on https://github.com/trinodb/trino/issues/10096 it is known it is not going working reliably. Once I read the code I was surprised it was failing that rarely. 

I would prefer to use `@Flaky` when it is not known why it fails, where according to our best knowledge it should be passing.